apps/laboratory/src/utils/WagmiConstants.ts

```diff
@@ -1,4 +1,4 @@
-import { defaultWagmiConfig } from '@web3modal/wagmi/react/config'
+import { defaultWagmiConfig } from '@web3modal/wagmi/react'
```

Comment:
We continue exporting `/react/config` path. Removed `/config` path since it's no required. But for users they could still continue using it.

Reply:
@glitch-txs remember why we needed to split this? it was a bundling issue IIRC